README.md

```diff
@@ -55,6 +61,15 @@ make clean
 
 Cleans the build directory.
 
+## Note
+
+Your are not dependent on the [MAKEFILE](Makefile).\
+For Example: Windows User can also execute the hugo CLI commands manually like:
```

Comment:
"Example" -> "example"!

"User" -> "user"

Also a windows user can still use `make` as well.

Reply:
Hi fritzmg,
i have edit your suggestions on my fork: https://github.com/fkaminski/docs/blob/master/README.md

What are the further steps now?

Reply:
I think you did your edits in your master branch - however, this PR is from your `patch-2` branch ;)